.github/codecov.yml

```diff
@@ -4,9 +4,9 @@
 codecov:
   notify:
     wait_for_ci: yes
-    after_n_builds: 4 # (unit_tests, bpf2c) * (Debug, Release)
+    after_n_builds: 12 # (reusable-build.yml instances in cicd.yml) * (Debug, Release)
 comment:
-    after_n_builds: 4 # (unit_tests, bpf2c) * (Debug, Release)
+    after_n_builds: 12 # (reusable-build.yml instances in cicd.yml) * (Debug, Release)
```

Comment:
Should it be `reusable-rest.yml` instances instead?

Reply:
I believe it is reusable-build.yml, because it is called 6 times in cicd.yml file for each Debug and the release will be 12 times in total. After-n-builds will delay after waiting for these 12 builds.

Reply:
Code coverage is reported by reusable-test, not reusable-build.

Reply:
Updated